The source fails to build because of the missing format.
Index: sm-ssc-5~alpha3/src/RageInput.cpp
===================================================================
--- sm-ssc-5~alpha3.orig/src/RageInput.cpp	2012-08-06 07:23:32.000000000 +0200
+++ sm-ssc-5~alpha3/src/RageInput.cpp	2012-09-23 14:38:51.867469713 +0200
@@ -66,7 +66,7 @@
 
 	// If no input devices are loaded, the user won't be able to input anything.
 	if( apDevices.size() == 0 )
-		LOG->Warn( NO_INPUT_DEVICES_LOADED.GetValue() );
+		LOG->Warn( "%s", NO_INPUT_DEVICES_LOADED.GetValue().c_str() );
 }
 
 void RageInput::Update()
